### PR TITLE
test: Cypress - Map chart automation

### PR DESCRIPTION
**Pull request by sarojsarab**
## Description
- Map Chart Automation
#### PR fixes following issue(s)
- Added tests for Map Chart Widget
#### Type of change
- Cypress
#### How Has This Been Tested?
- [x] Cypress
>
>

#### QA activity:
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed

### CODE REVIEW

**Comment by sarojsarab** [comment]
Test run : https://github.com/appsmithorg/appsmith/actions/runs/6704111356

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/Others/MapChart_Spec.ts [hunk 1]
```diff
@@ -0,0 +1,221 @@
+/// <reference types="Cypress" />
+const viewWidgetsPage = require("../../../../../locators/ViewWidgets.json");
+import * as _ from "../../../../../support/Objects/ObjectsCore";
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/Others/MapChart_Spec.ts:3]: Remove this import.

**Review by Aishwarya-U-R — COMMENTED** [review]
@sarojsarab Script looks fine. Are all tests from removed .js spec file covered in new spec?
Please once confirm on its coverage from QA. And Are we going to run it in all PR or Hosted runs?

**Comment by Aishwarya-U-R** [comment]
[Fix lint](https://github.com/appsmithorg/appsmith/actions/runs/6704421810/job/18216808144?pr=28512)

**Comment by Aishwarya-U-R** [comment]
> Test run : https://github.com/appsmithorg/appsmith/actions/runs/6704111356

Run it few more time before PR can be merged.

**Comment by sarojsarab** [comment]
> > Test run : https://github.com/appsmithorg/appsmith/actions/runs/6704111356
> 
> Run it few more time before PR can be merged.

Sure

**Comment by sarojsarab** [comment]
> [Fix lint](https://github.com/appsmithorg/appsmith/actions/runs/6704421810/job/18216808144?pr=28512)

Done

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/Others/MapChart_Spec.ts (hunk 1 above)
Comment (sarojsarab) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/Others/MapChart_Spec.ts]: Done, thanks for pointing this out.

**Comment by sarojsarab** [comment]
> @sarojsarab Script looks fine. Are all tests from removed .js spec file covered in new spec? Also please once confirm on its coverage from QA. And Are we going to run it in all PR or Hosted runs?

Yes, All the tests from the JS are covered here. It doesn't have dependency on any authorisation and all so can be run as part of all the pr's.

Will Confirm the coverage from QA and let you know.

**Comment by sarojsarab** [comment]
> > It doesn't have dependency on any authorisation
> 
> Got it, no dependency on map api key also right?

Yes no dependency at all

**Review by Aishwarya-U-R — APPROVED** [review]

**Pull request merged**